model/registry/messaging.yaml

```
@@ -30,6 +40,15 @@ groups:
           the broker doesn't have such notion, the destination name SHOULD uniquely identify the broker.
         examples: ['MyQueue', 'MyTopic']
         tag: messaging-generic
+      - id: destination.subscription.name
+        type: string
+        stability: experimental
+        brief: 'The name of the topic subscription message is consumed from.'
```

```suggestion
        brief: 'The name of the destination subscription from which a message is consumed.'
```

Maybe we should avoid the "topic" terminology?